src/silx/gui/plot/StackView.py

```diff
@@ -56,7 +56,7 @@
 
 
     sv = StackViewMainWindow()
-    sv.setColormap("jet", autoscale=True)
+    sv.setColormap("viridis", vmin=numpy.min(mystack), vmax=numpy.max(mystack))
```

Comment:
Maybe safer to use `minmax` (for nan/inf pbm)

Reply:
jet hihihi

Reply:
Yes, but that's for sample code, then I would hard-code the value.

Reply:
Aaaaoutch, that's a big bunch of code for the documentation...